capeinfra/swimlanes/private.py

```diff
@@ -169,20 +176,29 @@ def create_dap_api(self):
             # TODO: ISSUE #65
             opts=ResourceOptions(
                 parent=self,
-                # NOTE: not specying these led to the deployment being
+                # NOTE: not specifying these led to the deployment being
                 #       constructed before things it depends on
                 depends_on=[post_new_dap_method, post_new_dap_integration],
             ),
         )
 
+        # NOTE: our stage name is in the config file, and if it is not defined
+        #       we really want the deployment to fail. so we'll let the
+        #       KeyError happen and not try to do anything about it
+        stage_name = (
+            self.get_config_dict()
+            .get("api")
+            .get("dap")
+            .get("meta")
+            .get("stage-name")
+        )
```

Comment:
```suggestion
        stage_name = self.config.get("api", "dap", "meta", "stage-name")
```

This will become much simpler after #81 , we should just  hold off on this PR I think until then